.github/workflows/release-desktop-app.yml

```diff
@@ -169,7 +175,17 @@ jobs:
         with:
           name: affine-linux-x64-builds
           path: ./
-
+      - name: Download Artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: release-yml-build-script
+          path: ./
+      - uses: actions/setup-node@v3
```

Comment:
may use ./.github/actions/setup-node instead

Reply:
in the final release step we only need a simple nodejs env, dont need to checkout repo or restore cache, so maybe we dont need to use the exist action